src/CoreFx.Private.TestUtilities/src/System/PlatformDetection.Unix.cs

```diff
@@ -67,6 +73,44 @@ public static string GetDistroVersionString()
             return "Distro=" + v.Id + " VersionId=" + v.VersionId;
         }
 
+        /// <summary>
+        /// If gnulibc is available, returns the release, such as "stable".
+        /// Otherwise returns "glibc_not_found".
+        /// </summary>
+        public static string LibcRelease
+        {
+            get
+            {
+                try
+                {
+                    return Marshal.PtrToStringUTF8(gnu_get_libc_release());
```

Comment:
Should this be saved to a field?

Reply:
done

Reply:
it's called exactly once, for telemetry

Reply:
@danmosemsft  should i revert it then ?

Reply:
My bad that must be done with a copy of the code since this is for tests. It's fine as you did it

Reply:
@danmosemsft i didnt understand ur comment ?